src/i965_drv_video.c

```diff
@@ -1078,6 +1078,27 @@ i965_GetConfigAttributes(VADriverContextP ctx,
 
             break;
 
+        case VAConfigAttribEncMaxSlices:
+            if (entrypoint == VAEntrypointEncSlice ||
+                entrypoint == VAEntrypointEncSliceLP) {
```

Comment:
As the VAConfigAttribEncMaxSlices is added at the first time, I am not sure the VAEntrypointEncSliceLP support/test multi slices. VAEntrypointEncSliceLP support the multi slice, right?